config/labels/analyzers/clang-tidy.json

```diff
@@ -1795,7 +1795,9 @@
     ],
     "clang-diagnostic-conditional-uninitialized": [
       "doc_url:https://clang.llvm.org/docs/DiagnosticsReference.html#wconditional-uninitialized",
-      "severity:MEDIUM"
+      "guideline:sei-cert",
```

Comment:
Add `profile:security` and maybe even `profile:default` for cases where we have a guideline to keep in sync with the rest of the checkers in profiles!